pytaxonkit.py

```diff
@@ -974,6 +984,14 @@ def test_lca_unfound(capsys):
     assert 'taxonkit lca --skip-unfound' in terminal.err
 
 
+@pytest.mark.parametrize("domulti, ids,result", [
+    (False, [775536, 2238728, 1121123211234321], None),
+    (True, [[1766280, 406491, 2568889], [11111111111, 20487, 760325]], [None, None])
+])
+def test_lca_all_missing(ids, domulti, result):
+    assert lca(ids, multi=domulti, skip_deleted=True, skip_unfound=True) == result
+
+
```

Comment:
New test to verify expected behavior when input consists entirely of deleted or unfound taxids.

Reply:
Note that if it's a multi-LCA query and the output of `taxonkit lca` is empty, we want a list of `None` rather than a single `None`.

Reply:
Can confirm returns expected outcome.